tasks/go.py

```diff
@@ -359,26 +359,29 @@ def reset(ctx):
 
 
 @task
-def check_go_mod_replaces(_):
+def check_go_mod_replaces(ctx, fix=False):
     errors_found = set()
     for mod in get_default_modules().values():
-        go_sum = os.path.join(mod.full_path(), "go.sum")
-        if not os.path.exists(go_sum):
-            continue
-        with open(go_sum) as f:
-            for line in f:
-                if "github.com/datadog/datadog-agent" in line.lower():
-                    err_mod = line.split()[0]
-
-                    if (Path(err_mod.removeprefix("github.com/DataDog/datadog-agent/")) / "go.mod").exists():
-                        errors_found.add(f"{mod.import_path}/go.mod is missing a replace for {err_mod}")
+        with ctx.cd(mod.path):
+            go_sum = os.path.join(mod.full_path(), "go.sum")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Do you still need `full_path` since you cd-ed inside the go mod path?

Reply:
I tried using just relative file `go.sum` and `./go.sum` and it read the top. level `go.sum` even after `ctx.cd`. I'm not fully sure if I was missing something, but leaving `os.path.join(mod.full_path(), "go.sum")` seems to work 😄 

Do you know an alternative?  Maybe `os.path.join(mod.path, "go.sum")`